include/libp2p/crypto/key_generator/key_generator_impl.hpp

```diff
@@ -34,7 +34,7 @@ namespace libp2p::crypto {
    private:
     void initialize();
 
-    outcome::result<KeyPair> generateRsa(common::RSAKeyType key_type) const;
+//    outcome::result<KeyPair> generateRsa(common::RSAKeyType key_type) const;
```

Comment:
I think from here it better be removed.

Reply:
It should be rewritten, we have pre-314 for this purpose.
It is mentioned in cpp file.